ansible/roles/bastion-lite/tasks/main.yml

```diff
@@ -0,0 +1,112 @@
+#vim: set ft=ansible:
+---
+# tasks file for bastion
+
+######################### Setting up environment for post deployment administration
+# These are already done in set_env_authorized_key role and will be removed
+# - name: create /root/.ssh
+#   file:
+#     dest: /root/.ssh
+#     mode: 0700
+#     state: directory
+#   when: not hostvars.localhost.skip_packer_tasks | d(false)
+#   tags: packer
+
+# - name: copy the environment .pem key
+#   become: true
+#   copy:
+#     src: "{{output_dir}}/{{ env_authorized_key }}"
+#     dest: /root/.ssh/{{env_authorized_key}}.pem
+#     owner: root
+#     group: root
+#     mode: 0400
+#   when: use_own_key|bool
+
+- name: copy the user's SSH private key
+  become: true
+  copy:
+    src: "~/.ssh/{{key_name}}.pem"
+    dest: "/root/.ssh/{{key_name}}.pem"
+    owner: root
+    group: root
+    mode: 0400
+  when: not use_own_key|bool
+  tags:
+    - copy_env_private_key
+
+- name: Generate host .ssh/config Template
+  become: no
+  local_action: template src={{ role_path }}/files/bastion_ssh_config.j2 dest={{output_dir}}/ssh-config-{{ env_type }}-{{ guid }}
+  tags:
+    - gen_sshconfig_file
+
+- name: copy over host .ssh/config Template
+  become: true
+  copy:
+    src: "{{output_dir}}/ssh-config-{{ env_type }}-{{ guid }}"
+    dest: /root/.ssh/config
+    owner: root
+    group: root
+    mode: 0400
+  tags:
+    - copy_sshconfig_file
+
+- name: Install python-requests
+  ignore_errors: yes
+  become: true
+  yum:
+    name:
+    - python-requests
+  when: not hostvars.localhost.skip_packer_tasks | d(false)
```

Comment:
Is packer relevant for OpenStack? @fridim

Reply:
Yes, will be.